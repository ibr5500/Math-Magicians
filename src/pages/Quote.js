import React from 'react';

const Quote = () => {
  return (
    <div className='quote'>
      <p>
        " <b>Mathematics</b> is not about numbers, equations, computations, or
        algorithms: it is about understanding. " --<b>William Paul Thurston</b>
      </p>
    </div>
  );
}

export default Quote;