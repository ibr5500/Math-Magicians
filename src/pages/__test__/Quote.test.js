import renderer from 'react-test-renderer';
import Quote from '../Quote';

describe('Test for the Quote page', () => {
    it('Should render the Quote page', () => {
      const tree = renderer.create(<Quote />);
      expect(tree).toMatchSnapshot();
    });
});