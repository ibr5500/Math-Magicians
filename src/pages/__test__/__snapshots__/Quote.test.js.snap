// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for the Quote page Should render the Quote page 1`] = `
<div
  className="quote"
>
  <p>
    " 
    <b>
      Mathematics
    </b>
     is not about numbers, equations, computations, or algorithms: it is about understanding. " --
    <b>
      William Paul Thurston
    </b>
  </p>
</div>
`;
